CHANGELOG.md

```diff
@@ -4,11 +4,19 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
```

Comment:
Should the "Unreleased" header be kept in this branch or is it added back later?

Reply:
i think i usually removed it and the first new entry after the release adds it back but no strong opinions either way but in the versioned branch it should be removed